From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 30 Aug 2024 00:22:14 -0400
Subject: [PATCH 5/5] Reorder dependencies

Debian doesn't handle range dependencies when the upper values in the
range aren't in debian, because the buildd network always selects the
first of a conjunction, and the versions are sorted in that order.

Forwarded: not-needed
---
 Cargo.toml | 2 +-
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index dff28a5..8562045 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -237,7 +237,7 @@
 version = "0.2.19"
 
 [dependencies.num_enum]
-version = ">=0.5.7, < 0.8"
+version = "0.5.7"
 
 [dependencies.ocb3]
 version = "0.1"
